### PR TITLE
Partially update user guide

### PR DESCRIPTION
**Pull request by wltan**
#131 
Updates user guide for commands on adding and editing expenses and incomes, along with a brief summary of expenses and incomes in general.

### CODE REVIEW

**Review by yongping827 — DISMISSED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,42 +145,75 @@ Switched to overview tab.
...
+New expense added: Bubble Tea Amount: $5.00 Date: 03/10/2020 Categories: [Food & Beverage]
+```
 
 #### 4.3.2 Edit Expense
 
-Edits an expense in the finance tracker.
+Edits an expense in the finance tracker. 
 
-Format:
+Format: (when on the [Expenses tab](#)) `edit INDEX [t/TITLE] [a/AMOUNT] [d/DATE] [c/CATEGORY]...`
```
Comment (wltan) [inline docs/UserGuide.md:193]: Link will be fixed when Tab section has been added.

image

File: docs/UserGuide.md [hunk 2]
```diff
@@ -244,42 +277,75 @@ Expected Outcome:
...
+and cannot begin with a space.
+- `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
+- `DATE` should be in `dd/mm/yyyy` format, and cannot be later than the current date.
+- `CATEGORY` is optional. Multiple `c/` prefixes can be used to specify multiple categories.
+Each category name should consist of <abbr title="Alphanumeric characters, space, and the special characters !&quot;#$%&'()*+,-./:;&lt;=&gt;?@[\]^_`{\|}~">printable ASCII characters</abbr>,
+and cannot begin with a space.
 
-* `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
-* `DATE` should be in `dd/mm/yyyy` format.
+Shortcut: `addi`, (when on the [Income tab](#)) `add`
```
Comment (wltan) [inline docs/UserGuide.md:303]: https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/178/files#r511513961

File: docs/UserGuide.md [hunk 3]
```diff
@@ -145,42 +145,75 @@ Switched to overview tab.
...
 
 Expected Outcome:
+```
+New expense added: Bubble Tea Amount: $5.00 Date: 03/10/2020 Categories: [Food & Beverage]
+```
 
 #### 4.3.2 Edit Expense
 
-Edits an expense in the finance tracker.
+Edits an expense in the finance tracker.
```
Comment (ianyong) [inline docs/UserGuide.md:191]:
```suggestion
Edits an expense in the finance tracker.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -244,42 +277,75 @@ Expected Outcome:
...
 
 Expected Outcome:
+```
+New income added: Internship Amount: $560.00 Date: 03/10/2020 Categories: [Work]
+```
 
 #### 4.4.2 Edit Income
 
-Edits an income in the finance tracker.
+Edits an expense in the finance tracker.
```
Comment (ianyong) [inline docs/UserGuide.md:323]:
```suggestion
Edits an expense in the finance tracker.
```

**Review by ianyong — CHANGES_REQUESTED** [review]

**Review by ianyong — APPROVED** [review]
LGTM!

**Review by yongping827 — APPROVED** [review]
LGTM

**Pull request merged**